src/Mod/Part/App/FaceMaker.h

```diff
@@ -77,9 +86,9 @@ class PartExport FaceMaker: public BRepBuilderAPI_MakeShape, public Base::BaseCl
     virtual const TopoDS_Face& Face();
 
 #if OCC_VERSION_HEX >= 0x070600
-    void Build(const Message_ProgressRange& theRange = Message_ProgressRange()) override;
+    virtual void Build(const Message_ProgressRange& theRange = Message_ProgressRange());
```

Comment:
Why the change from `override` to `virtual`? Isn't this overriding a base class method?